src/main/java/com/fasterxml/jackson/databind/AnnotationIntrospector.java

```diff
@@ -68,7 +68,6 @@ public enum Type {
              * {@link com.fasterxml.jackson.annotation.JsonBackReference}
              */
             BACK_REFERENCE
-            ;
```

Comment:
Stylistically I kind of like ending semicolon, but since this is like third time someone commits such change, ok. I'll get rid of them.

Reply:
My personal style tends to the "less is more" and it just something that keeps coming up in my IDE highlighting. Not any functional impact at all, i agree.

I'm fine with whatever the core team feels is best – if decided in that way it could always be added to the https://github.com/FasterXML/jackson/blob/master/contribution/jackson-coding-style-guide.md, and i'd configure my IDE hints accordingly.